src/AutoRest.CSharp/Common/Input/CodeModelConverter.cs

```diff
@@ -380,7 +380,7 @@ private static InputType CreateType(Schema schema, string? format, IReadOnlyDict
             Namespace: schema.Extensions?.Namespace,
             Accessibility: schema.Extensions?.Accessibility,
             Description: schema.CreateDescription(),
-            Usage: InputModelTypeUsage.RoundTrip,
+            Usage: InputModelTypeUsage.None,
```

Comment:
why do we change to `None`?

Reply:
It is from swagger, and there is no need to generate enum, so the Usage is None